### ISSUE TITLE
bg-prov template-v-1 creates binary instead of JSON

### ISSUE THREAD

**Issue by ansiwen** [9elements/converged-security-suite#368]
```
$ ./bg-prov template-v-1 --ibbhash=SHA256 config.json
$ xxd config.json 
00000000: 5f5f 4143 4250 5f5f 1000 0000 0000 0000  __ACBP__........
00000010: 5f5f 4942 4253 5f5f 1000 0000 0000 0000  __IBBS__........
00000020: 0000 0000 0000 0000 0000 0000 0000 0000  ................
00000030: 0000 0000 0000 0000 0000 0000 0000 0000  ................
00000040: 0000 0000 0000 0000 0b00 0000 0000 0b00  ................
00000050: 0000 005f 5f50 4d53 475f 5f10 1000 0010  ...__PMSG__.....
00000060: 0000 0000 1000 0000 00                   .........
```

**Comment by zaolin** [comment]
Related to #355

**Comment by walterchris** [comment]
Resolved with #355 .
